Python/qled/qled.py

```diff
@@ -0,0 +1,129 @@
+from PySide6.QtWidgets import QAbstractButton
+from PySide6.QtCore import QTimer, Qt, QByteArray
+from PySide6.QtGui import QColor, QPainter
+from PySide6.QtSvg import QSvgRenderer
+import importlib.resources
+from pathlib import Path
+from enum import Enum
+
+
+class QLed(QAbstractButton):
+
+    class ledShape(Enum):
+        Circle = "circle.svg"
+        Square = "square.svg"
+        Triangle = "triangle.svg"
+        Rounded = "rounded.svg"
+        Rectangle = "rect.svg"
+
+    def __init__(self, parent=None, shape=ledShape.Circle):
+        super().__init__(parent)
+
+        self._value = False
+        self._blink = False
+        self._colorOn = QColor("green")
+        self._colorOff = QColor("gray")
+
+        self._checkable = True
```

Comment:
This seems like a leftover from the previous version where we did only inhert from `QWidget`, because `QAbstractButton` already comes with the whole "checkable" functionality.